prefect_databricks/credentials/__init__.py

```diff
@@ -0,0 +1,5 @@
+from ._generated import DatabricksCredentials as _DatabricksCredentials
+
+
+class DatabricksCredentials(_DatabricksCredentials):
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5GTHI1PH2dTiantfps6Fnc/1c750fab7f4c14ea1b93a62b9fea6a94/databricks_logo_icon_170295.png?h=250"  # noqa
```

Comment:
I think we can set `_logo_url` in this to avoid changing the inheritance chain.
```suggestion
DatabricksCredentials._logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5GTHI1PH2dTiantfps6Fnc/1c750fab7f4c14ea1b93a62b9fea6a94/databricks_logo_icon_170295.png?h=250"  # noqa
```